thresher/models.py

```diff
@@ -65,11 +65,20 @@ class Topic(models.Model):
     # an id within the given Analysis Type
     topic_id = models.IntegerField() 
 
+    # an id of its parent topic
+    parent_id = models.ForeignKey(self, related_name="subtopics")
+
     # The analysis type to which this topic belongs
-    analysis_type = models.ForeignKey(AnalysisType, related_name='topics')
+    # analysis_type = models.ForeignKey(AnalysisType, related_name='topics')
 
     # The name of the topic
     name = models.TextField()
+
+    # Glossary for analysis, copied from Analysis Type
+    glossary = models.TextField() # as a JSON map
```

Comment:
Two spaces before "#" according to PEP8 Python Style guide (https://www.python.org/dev/peps/pep-0008/)